msl_expressions/autogenerated/src/Plans/GenericStandards/StandardAlignAndGrab2Receivers.cpp

```diff
@@ -341,6 +354,9 @@ namespace alica
         this->trans = (*sc)["Behaviour"]->get<double>("StandardAlign.AlignSpeed", NULL);
         string tmp;
         bool success = true;
+        this->canPass = true;
+        this->canPassCounter = 1;
+        this->canPassThreshold = 1;
         try
```

Comment:
I think this somewhat improved pass decisions during the tournament. Might be improved to remove "hack" status, so merging and opening ticket is probably best.